src/config.ts

```diff
@@ -1,4 +1,8 @@
 import { MessageEmbed } from 'discord.js';
+import dotenv from 'dotenv';
+
+// Use dotenv to find system variables or a .env file.
+dotenv.config();
```

Comment:
I get the following error:

```bash
eddiebot-nodejs_1  | TypeError: Cannot read property 'config' of undefined
eddiebot-nodejs_1  |     at Object.<anonymous> (/opt/EddieBot/src/config.ts:5:8)
eddiebot-nodejs_1  |     at Module._compile (internal/modules/cjs/loader.js:1076:30)
eddiebot-nodejs_1  |     at Module._compile (/opt/EddieBot/node_modules/source-map-support/source-map-support.js:547:25)
eddiebot-nodejs_1  |     at Module.m._compile (/tmp/ts-node-dev-hook-4995269187454796.js:57:33)
eddiebot-nodejs_1  |     at Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
eddiebot-nodejs_1  |     at require.extensions.<computed> (/tmp/ts-node-dev-hook-4995269187454796.js:59:20)
eddiebot-nodejs_1  |     at Object.nodeDevHook [as .ts] (/opt/EddieBot/node_modules/ts-node-dev/lib/hook.js:63:13)
eddiebot-nodejs_1  |     at Module.load (internal/modules/cjs/loader.js:941:32)
eddiebot-nodejs_1  |     at Function.Module._load (internal/modules/cjs/loader.js:782:14)
eddiebot-nodejs_1  |     at Module.require (internal/modules/cjs/loader.js:965:19)
eddiebot-nodejs_1  | [ERROR] 07:36:04 TypeError: Cannot read property 'config' of undefined
```

I think on the import on live `5`, it should be changed from

```ts
import * as dotenv from 'dotenv';
```

to

```ts
import * as dotenv from 'dotenv';
```

Reply:
It should yea, this is an ES6 project!